### ISSUE TITLE
Run Options

### ISSUE THREAD

**Issue by jdoleary** [jdoleary/Spellmasons#758]
- Exclude certain enemies
- Exclude spell categories ("Beat the game with no damage spells" could be fun)
- Change starting stats (of NPCs too??)
- what else?

**Comment by SoulMuncher** [comment]
Would tie in with runes really well, you could enable some from the start as "run modifiers"

"Perfectionist" - All incoming damage is lethal or something like that

**Comment by jdoleary** [comment]
Yeah I'm starting to get a grasp for how Runes could work actually.  It's almost as if classes themselves could just be runes and when I think about them stacking the possibilities expand.

Like a rune that multiplies all near damage would be cool.
Or one that increases damage for as much stamina as you have or something.  Lots of possibilities here.
